docs/users_guide/ocean/test_groups/global_ocean.rst

```diff
@@ -305,14 +305,30 @@ ocean domain.
 A variant of this mesh has been used for low resolution simulations as part of
 the `E3SM v1 Cryosphere Campaign <https://e3sm.org/research/cryosphere-ocean/v1-cryosphere-ocean/>`_.
 
+.. _global_ocean_mesh_kuroshio:
+
+Kuroshio8to60 and Kuroshio12to60
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Kuroshio 8- and 12- to 60-km meshes (Kuroshio8to60 and Kuroshio12to60) are
+designed to explore dynamics of the Kuroshio Current.
+
+The meshes have 8 and 12 km resolution, respectively, in the eastern
```

Comment:
eastern -> western

Reply:
@xylar Thank you for running the initial testing (including dynamical adjustments) for both simulations. I have checked the mesh information, streams, and namelist for ocean simulations. I think the cell number slightly differs perhaps due to the version difference (compass, JIGSAW). My mesh, initial conditions are based on compass from June 2022.

Here is the mesh etc. information comparison for the Kuroshio12to60 (I will post for Kuroshio8to60 in a separate comment). 38 cells less in my mesh. The cell distribution maps (cellWidthGlobal.png) look the same so not much of differences.

```
base_mesh.nc (Xylar)
dimensions:
        nCells = 431449 ;
        nEdges = 1294341 ;
        nVertices = 862894 ;
        maxEdges = 8 ;
        maxEdges2 = 16 ;
        TWO = 2 ;
        vertexDegree = 3 ;
        
base_mesh.nc (Yohei, mesh I am using for ocean biogeochemistry simulations)
dimensions:
        nCells = 431411 ;
        nEdges = 1294227 ;
        nVertices = 862818 ;
        maxEdges = 8 ;
        maxEdges2 = 16 ;
        TWO = 2 ;
        vertexDegree = 3 ;
```

I will also list the difference in the namelist options (major ones), submesoscale eddies are also turned on in my actual run but didn't include here. It is more about dt, del2 etc.

```
           Xylar's Test                       Yohei's BGC Run
 config_dt = '00:03:00'     config_dt = '00:10:00'
 (split explicit)
 config_hmix_ref_cell_width = 30.0e3     config_hmix_ref_cell_width = 20.0e3
 config_mom_del2 = 250.0      config_mom_del2 = 200.0
 config_mom_del4 = 1.5e10     config_mom_del4 = 1.18e10

 config_Redi_constant_kappa = 600.0      config_redi_constant_kappa = 400.0
 config_Redi_maximum_slope = 0.3      config_redi_maximum_slope = 0.01
```

I will compare the temperature and salinity distributions soon (**output.nc**?) with my test and runs.

Reply:
Here is the mesh etc. information comparison for the Kuroshio8to60. 404 cells less in my mesh. Again the cell distribution maps (cellWidthGlobal.png) look the same.

```
base_mesh.nc: (Xylar)
dimensions:
        nCells = 595675 ;
        nEdges = 1787019 ;
        nVertices = 1191346 ;
        maxEdges = 8 ;
        maxEdges2 = 16 ;
        TWO = 2 ;
        vertexDegree = 3 ;

base_mesh.nc: (Yohei, mesh I am using for ocean biogeochemistry simulations)
dimensions:
        nCells = 596271 ;
        nEdges = 1788807 ;
        nVertices = 1192538 ;
        maxEdges = 8 ;
        maxEdges2 = 16 ;
        TWO = 2 ;
        vertexDegree = 3 ;
```

```
         Xylar's Test                       Yohei's BGC Run
 config_dt = '00:03:00'     config_dt = '00:5:00'
 (split explicit)
 config_hmix_ref_cell_width = 30.0e3     config_hmix_ref_cell_width = 20.0e3
 config_mom_del2 = 250.0      config_mom_del2 = 25.0
 config_mom_del4 = 1.5e10     config_mom_del4 = 3.2e09

 config_Redi_constant_kappa = 600.0      config_redi_constant_kappa = 400.0
 config_Redi_maximum_slope = 0.3      config_redi_maximum_slope = 0.01
```

Reply:
Again I will compare the initial conditions and some of my test E3SM runs soon. Please let me know if I missed any information necessary, this is my first PR.

Reply:
```suggestion
The meshes have 8 and 12 km resolution, respectively, in the western
```

Reply:
@ytakano3, those del2 and del4 settings are important ones to correct.  Likely the Redi ones, too.  The difference in time step may or may not matter but I'll change that to be consistent.

I'll need to rerun everything after those changes so please hold off on looking at the results in more detail until I do that.